README.md

```diff
@@ -1,42 +1,179 @@
-micropython-lib
-===============
-micropython-lib is a project to develop a non-monolothic standard library
```

Comment:
I love how you here remove content written by me, just to reformat it and make look as written by you.